### ISSUE TITLE
Getting an error message while booting VM.

### ISSUE THREAD

**Issue by oliwerhelsen** [reddingwebpro/cognacbox#3]
```
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "618533d2-5810-46d0-abf4-937785e5d037", "--type", "headless"]

Stderr: VBoxManage: error: RawFile#0 failed to create the raw output file C:/Users/jason_06tt1g7/coding-public/newBox/ubuntu-bionic-18.04-cloudimg-console.log (VERR_FILE_NOT_FOUND)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

**Comment by wucherpfennig** [comment]
the following helped for me

```
    config.vm.provider "virtualbox" do |v|
        v.memory = 4096
        v.cpus = 2
        v.customize [ "modifyvm", :id, "--uartmode1", "disconnected" ] # THIS LINE
    end
```

**Comment by Franky1** [comment]
> the following helped for me

Unfortunately, the proposed solution has not helped me either.
The error message doesn't appear anymore, but the VM remains in the boot process forever...

**Comment by wucherpfennig** [comment]
@Franky1 

well I rember I had the same issue hence I updated the timeout. AND YES the machine takes ages to boot but after that everything runs smoothly. attached my modified vagrant file. (you do not need the swap config in the end. I needed this to run composer.)

Maybe this helps.

BR wucherpfennig

```
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|

    config.vm.boot_timeout = 1800 # 30 minutes

    config.vm.box = "reddingwebpro/cognacbox"
    config.vm.hostname = "cognacbox"
    config.vm.network "forwarded_port", guest: 80, host: 8080
    config.vm.network "private_network", ip: "192.168.33.10"
    config.vm.synced_folder ".", "/var/www", :mount_options => ["dmode=777", "fmode=777"]
    #config.vm.synced_folder ".", "/var/www", :nfs => { :mount_options => ["dmode=777","fmode=666","nolock","vers=3","udp"] }

    config.vm.provider "virtualbox" do |v|
        v.memory = 4096
        v.cpus = 2
        v.customize [ "modifyvm", :id, "--uartmode1", "disconnected" ]
    end

    config.vm.provision "shell", inline: "/bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=4096", run: "always"
    config.vm.provision "shell", inline: "/sbin/mkswap /var/swap.1", run: "always"
    config.vm.provision "shell", inline: "/sbin/swapon /var/swap.1", run: "always"

end

```

**Comment by reddingwebpro** [comment]
Testing a fix for this now. Hoping to release it tomorrow.

**Comment by reddingwebpro** [comment]
Source of the problems seems to be related to https://github.com/hashicorp/vagrant/issues/10578 -- implementing a patch in the code, and should be available shortly.

**Comment by reddingwebpro** [comment]
Fixed in v2.2 -- updated master.
